### PR TITLE
config timezone to 'Paris'

### PR DESCRIPTION
**Pull request by guillaumecabanel**
@cveneziani can you help me on where do I have to add `Date.current`?

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 module SeazonAtDojo
   class Application < Rails::Application
+    config.time_zone = 'Paris'
```
Comment (cveneziani) [inline config/application.rb]: Maybe put it below the generators block. We might not see it here.

**Review by cveneziani — APPROVED** [review]
That's a good start. 

Now you have to make sure that any call to date / time should use the time zone. Also make sure that your SQL queries are doing what you want them to do (eg. watch out for `beginning_of_day`)

**Comment by guillaumecabanel** [comment]
I think we are good:
```
[1] pry(#<Admin::StatisticsController>)> @begin_date
=> Wed, 15 Aug 2018
[2] pry(#<Admin::StatisticsController>)> @begin_date.beginning_of_day
=> Wed, 15 Aug 2018 00:00:00 CEST +02:00
[3] pry(#<Admin::StatisticsController>)> @begin_date.beginning_of_day.utc
=> 2018-08-14 22:00:00 UTC
[4] pry(#<Admin::StatisticsController>)> @begin_date.end_of_day
=> Wed, 15 Aug 2018 23:59:59 CEST +02:00
```

**Comment by cveneziani** [comment]
Not sure for these ones:

- https://github.com/guillaumecabanel/seazon_at_dojo/blob/master/app/controllers/admin/statistics_controller.rb#L21-L25
- https://github.com/guillaumecabanel/seazon_at_dojo/blob/master/app/models/serving.rb#L5-L10

**Comment by guillaumecabanel** [comment]
thank you for spotting this. I will handle these when I'll be comfortable with RSpec to test it.

**Pull request merged**